server/modules/user/repository/userRepository.js

```diff
@@ -75,23 +75,45 @@ export const countUsersBySurveyId = async (surveyId, countSystemAdmins = false,
     (row) => Number(row.count)
   )
 
-export const fetchUsers = async ({ offset = 0, limit = null }, client = db) =>
-  client.map(
-    `
-    SELECT ${selectFieldsCommaSep}
+const _usersSelectQuery = ({ selectFields, sortBy = 'email', sortOrder = 'ASC' }) => {
+  // check sort by parameters
+  const orderByFieldBySortBy = {
+    email: 'email',
+    name: 'name',
+    last_login_time: 'status, last_login_time', // 'status' is used to group users that have accepted invitation, otherwise they have never logged in
+    status: 'status',
+  }
+  const orderBy = orderByFieldBySortBy[sortBy] || 'email'
+  const orderByDirection = sortOrder && sortOrder.toUpperCase() === 'ASC' ? 'ASC' : 'DESC'
+
+  return `
+    WITH us AS (
+      SELECT DISTINCT ON (us.sess #>> '{passport,user}')
+        (us.sess #>> '{passport,user}')::uuid AS user_uuid,
+        (us.expire - interval '30 days') AS last_login_time
+      FROM user_sessions us
+      WHERE us.sess #>> '{passport,user}' IS NOT NULL
```

Comment:
Maybe session is better than `sess`?

Reply:
eheheh user_sessions is the table used by node-connect-pg-simple... we cannot rename the columns I think.